Explain 64-bit I-JSON issues but don't embed that in CDDL (#388)
quicwg/qlog: draft-ietf-quic-qlog-main-schema.md

defining new CDDL types for int64 and uint64 that limit their values to the
restricted 64-bit integer range. However, some of the protocols that qlog is
intended to support (e.g., QUIC, HTTP/3), can use the full range of uint64
values.

As such, to support situations where I-JSON is in use, seralizers MAY encode

> uint64 values using JSON strings. JSON parsers, therefore, SHOULD support

Reviewer: ```suggestion
  uint64 values using JSON strings. qlog parsers, therefore, SHOULD support
```

Not entirely sure that covers the entire load, but `JSON parsers` is also not quite correct I'd say :)

> parsing of strings or numbers into uint64 unless there is out-of-band

Reviewer: ```suggestion
parsing of uint64 values from JSON strings or JSON numbers unless there is out-of-band
```

> information indicating that neither the serializer or parser are constrained by

Reviewer: ```suggestion
information indicating that neither the serializer nor parser are constrained by
```
